src/js/view/controlbar.js

```diff
@@ -224,7 +226,7 @@ define([
 
             if(this.elements.playlist) {
                 this.elements.playlist.on('select', function (value) {
-                    this._model.once('setItem', function () {
+                    this._model.once('itemReady', function () {
```

Comment:
We should add a comment to clean this up. There's not a reliable way to remove the event. Ideally we could just call load and play in one command or load followed by play() would do the queueing reliably like autostart does above.